pinot-controller/src/main/java/org/apache/pinot/controller/recommender/io/InputManager.java

```diff
@@ -539,7 +528,7 @@ public int getNumKafkaPartitions() {
   }
 
   public boolean isIndexableDim(String colName) {
```

Comment:
Delete this method. It is not being used anywhere. If we need, we will add it back